internal/controllers/bundledeployment/bundledeployment.go

```diff
@@ -269,6 +270,12 @@ func (c *controller) reconcile(ctx context.Context, bd *rukpakv1alpha1.BundleDep
 			Reason:  rukpakv1alpha1.ReasonErrorGettingClient,
 			Message: err.Error(),
 		})
+		meta.SetStatusCondition(&bd.Status.Conditions, metav1.Condition{
```

Comment:
Copying this over from Slack:

Please add a singular condition - ObjectsHealthy - and the the conditions utilities to roll multiple other conditions up into a Ready or Healthy meta-condition. Having to manually set this every other time any other condition is set is fragile and error-prone.

Reply:
> and the the conditions utilities to roll multiple other conditions up into a Ready or Healthy meta-condition.

I don't disagree with the premise of having a rollup condition, but I feel like it might be out of scope for this PR.

> Having to manually set this every other time any other condition is set is fragile and error-prone.

+1 

Is the following a reasonable compromise?
- Make a function along the lines of `setInstalledAndHealthyFalse()` with whatever parameters are necessary, and call that instead of duplicating all of these lines.
- Make a follow-on issue to do a rollup, and/or figure out a way for Healthy to be fully orthogonal from Installed. (right now, and according to the RFC, if Installed=False, then Healthy must =False)